packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -663,52 +663,39 @@ export class BlockNoteEditor<
     ISchema,
     SSchema
   > {
-    const { node, depth, startPos, endPos } = getBlockInfoFromPos(
+    const { depth, blockContainer } = getBlockInfoFromPos(
```

Comment:
let's add unit tests for the `getTextCursorPosition` and `setTextCursorPosition` methods